CondFormats/PCLConfig/interface/AlignPCLThresholds.h

```diff
@@ -0,0 +1,58 @@
+#ifndef _AlignPCLThresholds_h_
+#define _AlignPCLThresholds_h_
```

Comment:
@mmusich I think this include guard should read ` CondFormats_PCLConfig_interface_AlignPCLThresholds_h` or similar